tomviz/python/tomviz/web.py

```diff
@@ -0,0 +1,167 @@
+import os
+import shutil
+import zipfile
+
+from paraview import simple
+from paraview.web.dataset_builder import ImageDataSetBuilder
+from paraview.web.dataset_builder import CompositeDataSetBuilder
+
+
+def web_export(destinationPath, exportType, deltaPhi, deltaTheta):
+    dest = '%s/data' % destinationPath
+    camera = {
+        'type': 'spherical',
+        'phi': range(0, 360, deltaPhi),
+        'theta': range(-deltaTheta, deltaTheta + 1, deltaTheta)
+    }
+
+    # Copy application
+    copy_viewer(destinationPath)
+
+    # Choose export mode:
+    if exportType == 0:
+        export_images(dest, camera)
+
+    if exportType == 1:
+        export_layers(dest, camera)
+
+    # Zip data directory
+    zipData(destinationPath)
+
+# -----------------------------------------------------------------------------
+# Helpers
+# -----------------------------------------------------------------------------
+
+
+def zipData(destinationPath):
+    dstFile = os.path.join(destinationPath, 'data.tomviz')
+    dataDir = os.path.join(destinationPath, 'data')
+    zf = zipfile.ZipFile(dstFile, mode='w')
```

Comment:
Probably better to use context manager here, then you don't need the try finally block, something like this

```python
with zipfile.ZipFile(dstFile, mode='w') as zf
   # Do stuff with zip here
```